grc/gui/Dialogs.py

```diff
@@ -176,6 +176,8 @@ def __init__(self, parent, message_type, buttons, title=None, markup=None,
             self, transient_for=parent, modal=True, destroy_with_parent=True,
             message_type=message_type, buttons=buttons
         )
+        self.set_keep_above(True)
+        self.set_transient_for(parent)
```

Comment:
`transient_for` is set in the super class constructor above. Is it needed here too?

Reply:
Oops , yeah it's not needed .
No, I tried all the other dialogs they all worked fine without it.